### ISSUE TITLE
anti-HLA class I antibodies

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7101]
**Preferred term label:**
anti-HLA class I antibodies

**Synonyms**
Anti-human leukocyte antigens (HLA) antibodies class I

**Definition (free text, please give PubMed ID)**
Presence of class I anti-HLA antibodies
29985463 [PMID] _23628391 [PMID] _33709433 [PMID] _ 28757117 [PMID]

**Parent term (use hpo.jax.org/app)**
NA

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Anti-HLA hyperimmunization ORPHA 2194

**Comment by pnrobinson** [comment]
added as Anti-class I HLA antibody positivity
def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against a class I human leukocyte antigen (HLA). Class I HLA corresponds to MHC class I (A, B, and C).
